src/Reducer.test.js

```diff
@@ -1,17 +1,18 @@
+import { combinedReducer, UPDATE_SELECTED_STOCK } from "./store";
 import {
-  combinedReducer,
   UPDATE_CHART_DATA,
-  UPDATE_CHART_RANGE,
-  UPDATE_COMPANY_OVERVIEW,
+  UPDATE_CHART_RANGE
+} from "./features/chart/redux/constants";
+import { UPDATE_COMPANY_OVERVIEW } from "./features/company/redux/constants";
```

Comment:
q: Wouldn't it be better to add these into the `index.js` file? You then wouldn't need to import the whole file path

Reply:
We will split up the test into feature folders, so no need to access these constant outside folders anymore.